### PR TITLE
Modernizing module

### PR DESCRIPTION
**Pull request by andreaswittig**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
```
Comment (michaelwittig) [inline package-lock.json:1]: is this file needed?

Comment (andreaswittig) [reply package-lock.json:1]: @michaelwittig As we are adding "dependencies" to some of the modules, I thought about adding a package-lock.json to all modules for consistency. Do you agree?

Comment (michaelwittig) [reply package-lock.json:1]: yes

**Review by michaelwittig — APPROVED** [review]
